src/vivarium_cluster_tools/psimulate/branches.py

```diff
@@ -129,16 +129,20 @@ def calculate_input_draws(
         existing draw numbers.
 
     """
+    MAX_DRAW_COUNT = 1000
+    if input_draw_count > MAX_DRAW_COUNT:
+        raise ValueError(f"Input draw count must be less than {MAX_DRAW_COUNT}.")
+
     np.random.seed(123456)
+    possible = list(range(MAX_DRAW_COUNT))
     if existing_draws:
-        possible = list(set(range(1000)).difference(existing_draws))
+        possible = list(set(possible).difference(existing_draws))
```

Comment:
Do we have guarantees that the set difference operation we do here will always give the same ordering of the resultant set? It might be worth avoiding sets altogether and just doing the slower list operations since it's only 1000 draws and we're only doing it once.

Reply:
Or you can `sorted(...)`

Reply:
Also, TIL that you can `set.difference(list)`

Reply:
I don't think this is an issue. When we do this operation, possible is ordered numerically increasing. The result of the difference is similarly ordered, minus those draws in the existing draws.

Reply:
I added the sorted call, even though it's likely superfluous.